content/training/Walk-through/clean_data.Rmd

```diff
@@ -348,23 +361,35 @@ linelist_cleaned <- linelist_cleaned %>%
+
+
 ### Cleaning patient origin  
 
-The variable `patient_origin` was originally the variable *quartier* in the raw dataset. **patient_origin is referenced in the place analyses chunks**, for example when calculating attack rates by region. In these steps, the variable `patient_origin` in `linelist_cleaned` is *joined to* the `patient_origin` variable of the data frame `population_data_region`, which was imported in the population and lab data steps.  
+The variable `patient_origin` (originally the variable *quartier* in the raw dataset) is referenced in the place analyses chunks, for example when calculating attack rates by region. In those steps, the variable `patient_origin` in `linelist_cleaned` is *joined to* the variable `patient_origin` of the data frame `population_data_region` (which was imported in the population and lab data steps).  
 
-**However, the place names in `population_data_region` are ALL CAPITAL LETTERS and this is not true in `linelist_cleaned` - the join will not register any matches.** The easiest way to solve this problem is to redefine the linelist_cleaned variable into all capital letters as well, using the base R function `toupper()`:  
+However, the place names in `population_data_region` are ALL CAPITAL LETTERS. This is not true in `linelist_cleaned` - so the join will not register any matches. The easiest way to solve this problem is to **redefine the `linelist_cleaned` variable into all capital letters as well**, using the base R function `toupper()`:  
 
 ```{r}
 linelist_cleaned$patient_origin <- toupper(linelist_cleaned$patient_origin)
 ```
 
-### Create lists of variables  
+### Create list variables  
 
 These variables names are stored in vectors that are created using the function `c()`. These vectors of names will be referenced in later code chunks. This way if you want to run the same function over these variables you can simply use the named vector rather than typing out each variable individually.  
 
-The default template code creates two vectors - one for symptoms variables (`SYMPTOMS`) and one for laboratory testing variables (`LABS`). However, *we can see that there are two rather different type of lab variables*: those with Positive/Negative values, and those with IgG and IgM results. To make our table of lab results more clean, we split the lab variables into these two groups.
-
-**Some variables do not apply to our Am Timan data and must be removed from the list or commented out (such as `convulsions` and `history of fever`).** If non-existant variables are left in the list it will cause errors in later code chunks.
+The default template code creates two vectors - one for symptoms variables (`SYMPTOMS`) and one for laboratory testing variables (`LABS`). **Some variables do not apply to our Am Timan data and must be removed from the list or commented out (such as `convulsions` and `history of fever`).** If non-existant variables are left in the list it will cause errors in later code chunks.
```

Comment:
We've updated the tab_linelist code to only give warnings if variables don't exist (see https://github.com/R4EPI/sitrep/issues/183)

```suggestion
The default template code creates two vectors - one for symptom variables (`SYMPTOMS`) and one for laboratory testing variables (`LABS`). 
```